### PR TITLE
Playwright tests for the new newsletter signup module

### PR DESCRIPTION
**Pull request by mmmavis**
Related PRs/issues: #11344

If you'd like to run the test locally, open a new terminal tab and run `npm run playwright:integration`

### CODE REVIEW

File: source/js/components/newsletter-signup/organisms/blog-body-signup.jsx [hunk 1]
```diff
@@ -198,7 +198,14 @@ class BlogBodySignForm extends Component {
 
   render() {
     return (
-      <div className="tw-relative tw-border tw-px-8 tw-pt-14 tw-pb-12 medium:tw-p-16 before:tw-absolute before:tw-top-0 before:tw-left-1/2 before:-tw-translate-x-1/2 before:-tw-translate-y-1/2 before:tw-content-[''] before:tw-inline-block before:tw-w-[72px] before:tw-h-14 before:tw-bg-[url('../_images/glyphs/letter.svg')] before:tw-bg-white before:tw-bg-no-repeat before:tw-bg-center before:tw-bg-[length:24px_auto]">
+      <div
+        className={`
+          ${this.props.innerWrapperClass}
```
Comment (mmmavis) [inline source/js/components/newsletter-signup/organisms/blog-body-signup.jsx:203]: This is the only change introduced to this file. The rest of the existing class names stay the same.

File: source/js/components/newsletter-signup/organisms/blog-body-signup.jsx [hunk 2]
```diff
@@ -198,7 +198,14 @@ class BlogBodySignForm extends Component {
...
+          tw-relative tw-border tw-px-8 tw-pt-14 tw-pb-12 medium:tw-p-16
+          before:tw-absolute before:tw-top-0 before:tw-left-1/2 before:-tw-translate-x-1/2 before:-tw-translate-y-1/2 before:tw-content-[''] before:tw-inline-block before:tw-w-[72px] before:tw-h-14 before:tw-bg-[url('../_images/glyphs/letter.svg')] before:tw-bg-white before:tw-bg-no-repeat before:tw-bg-center before:tw-bg-[length:24px_auto]
+          `}
+        data-submission-status={this.props.apiSubmissionStatus}
```
Comment (mmmavis) [inline source/js/components/newsletter-signup/organisms/blog-body-signup.jsx:207]: this attribute is newly introduced

**Review by jhonatan-lopes — APPROVED** [review]
LGTM :+1: awesome to see the env vars getting corrected on our CI as well!

**Comment by mmmavis** [comment]
🎉

**Pull request merged**